holoviews/operation/element.py

```diff
@@ -575,7 +575,7 @@ class decimate(ElementOperation):
        The x_range as a tuple of min and max y-value. Auto-ranges
        if set to None.""")
 
-    def _process(self, element, key=None):
+    def _apply(self, element, key=None):
```

Comment:
How about ``_process_layer`` then? At least this is supposed to be temporary, but best pick a meaningful name in case it isn't as temporary as we would like...